tests/topotests/bgp_default_originate/test_bgp_default_originate_topo1_3.py

```diff
@@ -14,8 +14,10 @@
 import os
 import sys
 import time
+from time import sleep
```

Comment:
Do we need this? I don't see it's used anywhere.

Reply:
fixed